corehq/apps/app_manager/suite_xml/sections/remote_requests.py

```diff
@@ -66,14 +66,14 @@ def case_session_var(self):
 
     def build_remote_request(self):
         return RemoteRequest(
-            post=self._build_remote_request_post(),
-            command=self._build_command(),
-            instances=self._build_instances(),
-            session=self._build_session(),
-            stack=self._build_stack(),
```

Comment:
Guessing this is what you were talking about on Slack the other day?  I personally prefer the underscore prefix, even if subclasses will be accessing the method.  I think of it as "clients of this class shouldn't use this method".  IMO there's unavoidably some leak between superclass and subclass, but it's handy to know what might be referenced in other parts of the codebase and what won't be.

Reply:
Yeah, it is. I think this can go either way - I like knowing that an underscore-prefixed method isn't used outside of the current file, which isn't guaranteed if subclasses have access - so I'm going to leave it as-is if we don't have a definitive team best practice.